packages/core/src/components/spinner/_spinner.scss

```diff
@@ -105,6 +105,11 @@ Styleguide components.progress.spinner.js.svg
   }
 }
 
+@keyframes pt-spinner-animation {
+  from { transform: rotate(0deg); }
+  to   { transform: rotate(360deg); }
+}
+
 $spinner-width: $pt-grid-size * 10 !default;
 $spinner-width-factor: 0.5 !default;
 $spinner-width-factor-small: 0.24 !default;
```

Comment:
I looked into simplifying the spinner CSS but it's a little more involved than I was ready for (mainly because most of the complexity is there to solve IE quirks)